src/main/java/org/mockbukkit/mockbukkit/sound/MusicInstrumentMock.java

```diff
@@ -36,6 +37,13 @@ public class MusicInstrumentMock extends MusicInstrument
 		return key;
 	}
 
+	@Override
+	public @NotNull String translationKey()
+	{
+		//TODO: Auto-generated method stub
+		throw new UnimplementedOperationException();
```

Comment:
I remember adding translation keys automatically to all json files, should be able to fetch this using MetaMiner if I'm not wrong of course